### DISCUSSION TITLE
GUI Freezes, There is no Output

### DISCUSSION THREAD

**Discussion by kakan18** [rafaelpadilla/review_object_detection_metrics#107] [Q&A]
Hi, we are trying to do a ship detection project, we used this repo for EfficientDet, Faster R-CNN and SSD MobileNet v2, but we could not make this work with YOLO.
![image](https://user-images.githubusercontent.com/62551738/171877088-452331d9-2041-4d1d-91b6-245dc88b83a7.png)
I looked it up, paths are correct. But still, I upload these screenshots for sample.
![image](https://user-images.githubusercontent.com/62551738/171877231-6ad1e1bd-9ea4-4114-a3f6-02c0791e6468.png)
There are 10.796 pictures in this file. We thought maybe it's okay that it freezes, but the other methods did not take this much time. Most of them was finished in 2-3 mins. This is going on for like 30 mins now.
![image](https://user-images.githubusercontent.com/62551738/171877570-2340d8ed-99e9-44fa-824a-c690665cb0ce.png)
![image](https://user-images.githubusercontent.com/62551738/171877674-9ea568f5-a025-48a8-8fdf-0161b946600b.png)
Here it is the command window, in case there is something wrong with it, I closed the app once then reopened that's why there are 2 run.py's.
![image](https://user-images.githubusercontent.com/62551738/171878203-8dd41453-179b-441d-bc6e-f57b515aea02.png)
I appriciate any help, thanks for even trying.
Thank you for this @rafaelpadilla , it made our project much easier to do!
 
**Update:**

It worked after like 50 minutes. I'm not sure why it happened, it may still worth to look at so I am not gonna delete it. I can try if you guys got suggestions, so thanks anyway!

**Comment by rafaelpadilla** [comment]
Hi @kakan18 ,

Thank you for your message!

We will investigate the reason for such a long time and try to improve speed up the processing time.

Is it possible to provide us the ~10.796 images + detections + groundtruths, so we can try to replicate the problem? If you don't feel comfortable to post them here, would it be possible to send me by email (rafael.padilla[at]smt.ufrj.br)? If you prefer, you could replace your original images by a complete blank image with the same resolution as the original.

I would like to measure the delay time and apply some improvements to see how much we can improve the processing time.

Best regards,
Rafael

**Reply by kakan18** [reply]
I just sent you an email, just writing here in case you didn't get notification.
